src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter((p) => p.sex === 'm' && Math.ceil(p.died / 100) === century)
+    : people.filter((p) => p.sex === 'm');
```

Comment:
Change the code to avoid repetition, but save readability.
Don’t shorten variables or function names. Check this [guide](https://mate-academy.github.io/style-guides/javascript.html#2311-dont-shorten-variables-or-functions-names).
```suggestion
  const men = people.filter((person) => ( 
    century 
      ? person.sex === 'm' 
      : person.sex === 'm' && Math.ceil(person.died / 100) === century)
    )
  )
```